internal/handlers/common/msg_listcommands.go

```diff
@@ -0,0 +1,188 @@
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// command represents a handler command.
+type command struct {
+	// Help is shown in the help function
+	Help string
+
+	// Handler processes command
+	Handler func(handlers.Interface, context.Context, *wire.OpMsg) (*wire.OpMsg, error)
+}
+
+// Commands is a map of Commands that Handler interface can support.
+// Order of entries matches the interface definition.
+//
+// Please keep help text in sync with handlers.Interface methods documentation.
+var Commands = map[string]command{
+	// sorted alphabetically
+	"buildInfo": {
+		Help:    "Returns a summary of the build information.",
+		Handler: (handlers.Interface).MsgBuildInfo,
+	},
+	"collStats": {
+		Help:    "Returns storage data for a collection.",
+		Handler: (handlers.Interface).MsgCollStats,
+	},
+	"connectionStatus": {
+		Help: "Returns information about the current connection, " +
+			"specifically the state of authenticated users and their available permissions.",
+		Handler: (handlers.Interface).MsgConnectionStatus,
+	},
+	"count": {
+		Help:    "Returns the count of documents that's matched by the query.",
+		Handler: (handlers.Interface).MsgCount,
+	},
+	"create": {
+		Help:    "Creates the collection.",
+		Handler: (handlers.Interface).MsgCreate,
+	},
+	"createIndexes": {
+		Help:    "Creates indexes on a collection.",
+		Handler: (handlers.Interface).MsgCreateIndexes,
+	},
+	"dataSize": {
+		Help:    "Returns the size of the collection in bytes.",
+		Handler: (handlers.Interface).MsgDataSize,
+	},
+	"dbStats": {
+		Help:    "Returns the statistics of the database.",
+		Handler: (handlers.Interface).MsgDBStats,
+	},
+	"debugError": {
+		Help:    "Returns error for debugging.",
+		Handler: (handlers.Interface).MsgDebugError,
+	},
+	"delete": {
+		Help:    "Deletes documents matched by the query.",
+		Handler: (handlers.Interface).MsgDelete,
+	},
+	"drop": {
+		Help:    "Drops the collection.",
+		Handler: (handlers.Interface).MsgDrop,
+	},
+	"dropDatabase": {
+		Help:    "Drops production database.",
+		Handler: (handlers.Interface).MsgDropDatabase,
+	},
+	"find": {
+		Help:    "Returns documents matched by the query.",
+		Handler: (handlers.Interface).MsgFind,
+	},
+	"findAndModify": {
+		Help:    "Inserts, updates, or deletes, and returns a document matched by the query.",
+		Handler: (handlers.Interface).MsgFindAndModify,
+	},
+	"getCmdLineOpts": {
+		Help:    "Returns a summary of all runtime and configuration options.",
+		Handler: (handlers.Interface).MsgGetCmdLineOpts,
+	},
+	"getLog": {
+		Help:    "Returns the most recent logged events from memory.",
+		Handler: (handlers.Interface).MsgGetLog,
+	},
+	"getParameter": {
+		Help:    "Returns the value of the parameter.",
+		Handler: (handlers.Interface).MsgGetParameter,
+	},
+	"hello": {
+		Help:    "Returns the role of the FerretDB instance.",
```

Comment:
Does it? I thought "ismaster" returns the role. Is this one an alias?

Reply:
`ismaster` and `isMaster` are aliases. `hello` is very similar, but slightly different. We tests for that: https://github.com/FerretDB/FerretDB/blob/main/integration/commands_replication_test.go